db/src/schema/user.rs

```diff
@@ -31,19 +32,32 @@ pub struct User {
     #[filter(insensitive, substring)]
     pub email: String,
     pub is_admin: bool,
+    pub is_active: bool,
```

Comment:
I think it's easier to add a `#[filter]` here instead of adding `UserExtraFilters`. The frontend would simply show the users page with `?is_active=true` and remove the filter if someone wants to list all users.
```suggestion
    #[filter]
    pub is_active: bool,
```

Reply:
I thought we might want the deactivated users to be omitted from the API response by default.
Do we? 
if not, then sure, it would be easier and I'll change it.

Reply:
Can't say I feel strongly about it, it just seems simpler and more symmetric in the API. We can leave it as-is.

Reply:
Fixed and pushed. Please review :)